src/processor/operator/persistent/reader/parquet/parquet_reader.cpp

```diff
@@ -674,11 +679,23 @@ static std::unique_ptr<function::TableFuncLocalState> initLocalState(
     return localState;
 }
 
+static double progressFunc(TableFuncSharedState* state) {
+    auto parquetScanSharedState = reinterpret_cast<ParquetScanSharedState*>(state);
+    if (parquetScanSharedState->fileIdx >= parquetScanSharedState->readerConfig.getNumFiles()) {
+        return 1.0;
+    }
+    if (parquetScanSharedState->totalRowsGroups == 0) {
+        return 0.0;
+    }
+    return (double)(parquetScanSharedState->blockIdx * (parquetScanSharedState->fileIdx + 1)) /
```

Comment:
ditto. Parquet parallelism is done the same way as parallel csv. So you need to change this as well.